doc/source/serve/getting_started.md

```diff
@@ -14,48 +25,43 @@ If you have suggestions on how to improve this tutorial,
 To run this example, you will need to install the following:
 
 ```bash
-$ pip install "ray[serve]" transformers
+$ pip install "ray[serve]" transformers requests
 ```
 
 
 ## Example Model
```

Comment:
Maybe rename this to "Example Model: Before Ray Serve" or similar?  As a new reader, I first glance at the first big code "Example" and it throws me off slightly that it doesn't have any Ray Serve code.

Reply:
I changed it to "Model Example: Before Ray Serve".